### PR TITLE
RAVTDBQ-455 Added flag for lower casing UDFs

### PR DESCRIPTION
**Pull request by rushikeshsapate-dm**
Added flag for Lower Casing User Defined Functions

### CODE REVIEW

File: core/src/main/java/org/apache/calcite/sql/SqlUtil.java [hunk 1]
```diff
@@ -304,7 +304,12 @@ public static void unparseFunctionSyntax(SqlOperator operator,
       }
       SqlIdentifier id = function.getSqlIdentifier();
       if (id == null) {
-        writer.keyword(operator.getName());
+        if (((SqlFunction) operator).getFunctionType() == SqlFunctionCategory.USER_DEFINED_FUNCTION
```
Comment (Mahesh-Raut) [inline core/src/main/java/org/apache/calcite/sql/SqlUtil.java:307]: Can you please add a simple comment over here as it only works, when we translate SQL from Mig.

Comment (rushikeshsapate-dm) [reply core/src/main/java/org/apache/calcite/sql/SqlUtil.java:309]: done

**Comment by Mahesh-Raut** [comment]
L1 Completed.

File: core/src/main/java/org/apache/calcite/sql/SqlUtil.java [hunk 2]
```diff
@@ -305,7 +305,13 @@ public static void unparseFunctionSyntax(SqlOperator operator,
       }
       SqlIdentifier id = function.getSqlIdentifier();
       if (id == null) {
-        writer.keyword(operator.getName());
+        // The following code block is executed exclusively when the code flow originates from mig.
+        if (((SqlFunction) operator).getFunctionType() == SqlFunctionCategory.USER_DEFINED_FUNCTION
```
Comment (sauajmera) [inline core/src/main/java/org/apache/calcite/sql/SqlUtil.java:309]: create a method and move this code .

private boolean shouldUnparseAsLowerCase(SqlFunction function, SqlWriter writer) {
    return function.getFunctionType() == SqlFunctionCategory.USER_DEFINED_FUNCTION &&
           writer.isUDFLowerCase();
}


also further convert code to ternary 

shouldUnparseAsLowerCase(function, writer)
        ? operator.getName().toLowerCase()
        : operator.getName()

**Comment by sauajmera** [comment]
changes looks good . L2 completed.

**Review by sauajmera — APPROVED** [review]
l2 completed

**Pull request merged**